.github/mergify.yml

```diff
@@ -41,6 +41,7 @@ pull_request_rules:
       - -conflict
       - -draft # filter-out GH draft PRs
       - label="0:] Ready-To-Go"
```

Comment:
```suggestion
```
@rohitgr7 shall we drop the label since it does not have any effect on the true merging?

Reply:
then we need to ensure that auto-merge should be enabled if a PR is ready to go else if it isn't then there is a possibility even after all the required approvals the PR is not merged. At least this still gives an indication that ok PR is good to go and one can then just directly merge it.

Reply:
well yes, we can return also the merge action to mergify, but still, I would make it on core responsibility to set the auto-merge when they feel that the PR is suitable to merge...

Reply:
ok my bad, not required here I guess.. I thought you are removing the label completely.